governance.md

```diff
@@ -130,6 +130,9 @@ open and be recorded for later viewing.
 
 Working groups are documented in [sigs.yaml](sigs.yaml).
 
+See [working group governance] for more details about forming and disbanding Working
```

Comment:
There is a lot of overlap here.  Might be good to just shrink this section and point to wg-governance.md.  Wouldn't be crazy to break out our other governance things into other docs over time.

Reply:
#2814